Plugins/Flow.Launcher.Plugin.WebSearch/SettingsControl.xaml

```diff
@@ -55,12 +70,12 @@
                 </GridView>
             </ListView.View>
         </ListView>
-        <StackPanel Grid.Row="3" HorizontalAlignment="Right" Orientation="Horizontal">
+        <StackPanel Grid.Row="4" HorizontalAlignment="Right" Orientation="Horizontal">
             <Button Click="OnDeleteSearchSearchClick" Width="100" Margin="10"
                     Content="{DynamicResource flowlauncher_plugin_websearch_delete}" />
             <Button Click="OnEditSearchSourceClick" Width="100" Margin="10"
                     Content="{DynamicResource flowlauncher_plugin_websearch_edit}" />
-            <Button Click="OnAddSearchSearchClick" Width="100" Margin="10"
+            <Button Click="OnAddSearchSearchClick" Width="100" Margin="10 10 0 10"
```

Comment:
Maybe add a margin to the right as well so that the button is not completely right aligned. 
```suggestion
            <Button Click="OnAddSearchSearchClick" Width="100" Margin="10 10 10 10"
```